app/component/ItineraryList/ItineraryList.js

```diff
@@ -140,89 +118,62 @@ function ItineraryList(
-              <div>
-                <FormattedMessage
-                  id="no-route-showing-alternative-options"
-                  defaultMessage="No routes with current settings found. Here are some alternative options:"
-                />
-              </div>
-            </div>
-          )}
-          {loadingMoreItineraries === 'top' && (
-            <div className="summary-list-spinner-container">
-              <Loading />
-            </div>
-          )}
-          {isBrowser && (
-            <div
-              className={cx('summary-list-items', {
-                'summary-list-items-loading-top':
-                  loadingMoreItineraries === 'top',
-              })}
-            >
-              {summaries}
-            </div>
-          )}
-          {loadingMoreItineraries === 'bottom' && (
-            <div className="summary-list-spinner-container">
-              <Loading />
-            </div>
-          )}
-          {isBrowser && canceledItinerariesCount > 0 && (
-            <CanceledItineraryToggler
-              showItineraries={showCancelled}
-              toggleShowCanceled={() => setShowCancelled(!showCancelled)}
-              canceledItinerariesAmount={canceledItinerariesCount}
-            />
-          )}
-        </div>
-        {onlyHasWalkingItineraries && !showAlternativePlan && (
-          <div className="summary-no-route-found" style={{ marginTop: 0 }}>
-            <div
-              className={cx('flex-horizontal', 'summary-notification', 'info')}
-            >
-              <Icon
-                className={cx('no-route-icon', 'info')}
-                img="icon-icon_info"
-                color="#0074be"
+      <div className="summary-list-container" role="list">
+        {showRelaxedPlanNotifier && (
+          <div
+            className={cx(
+              'flex-horizontal',
+              'summary-notification',
+              'show-alternatives',
+            )}
+          >
+            <Icon className="icon-icon_settings" img="icon-icon_settings" />
+            <div>
+              <FormattedMessage
+                id="no-route-showing-alternative-options"
+                defaultMessage="No routes with current settings found. Here are some alternative options:"
               />
-              <div>
-                <div className="in-the-past">
-                  <FormattedMessage
-                    id="router-only-walk-title"
-                    defaultMessage=""
-                  />
-                </div>
-                <FormattedMessage
-                  id="router-only-walk"
-                  defaultMessage={
-                    'Unfortunately no routes were found for your journey. ' +
-                    'Please change your origin or destination address.'
-                  }
-                />
-              </div>
-            </div>{' '}
+            </div>
+          </div>
+        )}
+        {loadingMore === 'top' && (
```

Comment:
Would be nice to explicitly know the possible values. We could use enum like solution for this.

Reply:
done